README.md

```diff
@@ -1,69 +1,105 @@
+ * A set of [Jenkins pipelines][piper-pipelines] using the piper library to
+   implement best practice processes.
 
 ## What you get
 
-This repository contains Jenkins pipeline instructions in form of a `Jenkinsfile` (together with some auxiliary files) which require the [shared library][piper-library] of project "Piper" to run. They are available under `pipelines/<scenario>/Jenkinsfile` in this repository.
+This repository contains Jenkins pipeline instructions in form of a
+`Jenkinsfile` (together with some auxiliary files) which require the [shared
+library][piper-library] of project "Piper" to run. They are available under
+`pipelines/<scenario>/Jenkinsfile` in this repository.
 
-These best practice Jenkinsfiles are based on the general concepts of [Pipelines as Code with Jenkins][jenkins-doc-pipelines]. They give you the power of the Jenkins community at hand to optimize your continuous delivery processes.
+These best practice Jenkinsfiles are based on the general concepts of [Pipelines
+as Code with Jenkins][jenkins-doc-pipelines]. They give you the power of the
+Jenkins community at hand to optimize your continuous delivery processes.
 
-You can run the Jenkinsfiles out of the box or take them as a starting point for project-specific adaptations.
+You can run the Jenkinsfiles out of the box or take them as a starting point for
+project-specific adaptations.
 
-## Installation
+# Requirements
 
-Prerequisites:
-
-* [Installed shared library][piper-library-installation] of project "Piper".
+* [Installed shared library][piper-library-readme] of project "Piper".
 * Jenkins User with privileges to create jobs.
 
-There are two recommended ways to consume our `Jenkinsfile`. For both ways, you need to create a new pipeline job in Jenkins that [retrieves a pipeline from an SCM][jenkins-doc-pipelineFromSCM].
-The location of the `Jenkinsfile` may differ:
+# Download and Installation
+
+There are two recommended ways to consume our `Jenkinsfile`. For both ways, you
+need to create a new pipeline job in Jenkins that [retrieves a pipeline from an
+SCM][jenkins-doc-pipelineFromSCM].  The location of the `Jenkinsfile` may
+differ:
 
-1. The `Jenkinsfile` can be copied into the source code repository of the application. In that case, no additional pipeline job parameters are required to retrieve the application source code repository.
+1. The `Jenkinsfile` can be copied into the source code repository of the
+   application. In that case, no additional pipeline job parameters are required
+   to retrieve the application source code repository.
 
-2. The `Jenkinsfile` can be kept in an own repository separate from the application. In case you want to apply the `Jenkinsfile` to multiple applications, this approach is beneficial since you only need to maintain the `Jenkinsfile` in one place. You must define two mandatory Jenkins pipeline job parameters in this case:
+2. The `Jenkinsfile` can be kept in an own repository separate from the
```

Comment:
My suggestion:
"The `Jenkinsfile` can be kept in its own repository, separate..."

Reply:
'its' -> Done

I see no reason for that comma, though.